paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/PaymentMethodMetadata.kt

```diff
@@ -32,6 +33,7 @@ internal data class PaymentMethodMetadata(
     val shippingDetails: AddressDetails?,
     val sharedDataSpecs: List<SharedDataSpec>,
     val hasCustomerConfiguration: Boolean,
+    val addressSchemas: AddressSchemas,
```

Comment:
Can you add a test for `AddressSchemas` like `SharedDataSpecParcelerTest`? I want to make sure we don't inadvertently break it.